tests/functional/spec/ukis/routing/why-no-innovation.spec.js

```diff
@@ -0,0 +1,459 @@
+  import PublicFinancialSupportForInnovationCompleted from '../../../pages/surveys/ukis/public-financial-support-for-innovation-completed.page.js'
+  import Turnover2014 from '../../../pages/surveys/ukis/turnover-2014.page.js'
+  import Turnover2016 from '../../../pages/surveys/ukis/turnover-2016.page.js'
+  import Exports2016 from '../../../pages/surveys/ukis/exports-2016.page.js'
+  import TurnoverAndExportsCompleted from '../../../pages/surveys/ukis/turnover-and-exports-completed.page.js'
+  import Employees2014 from '../../../pages/surveys/ukis/employees-2014.page.js'
+  import Employees2016 from '../../../pages/surveys/ukis/employees-2016.page.js'
+  import EmployeesQualifications2016 from '../../../pages/surveys/ukis/employees-qualifications-2016.page.js'
+  import EmployeesInHouseSkills from '../../../pages/surveys/ukis/employees-in-house-skills.page.js'
+  import EmployeesAndSkillsCompleted from '../../../pages/surveys/ukis/employees-and-skills-completed.page.js'
+  import AdditionalComments from '../../../pages/surveys/ukis/additional-comments.page.js'
+  import HowLong from '../../../pages/surveys/ukis/how-long.page.js'
+  import ApproachedTelephone from '../../../pages/surveys/ukis/approached-telephone.page.js'
+  import ReadyToSubmitCompleted from '../../../pages/surveys/ukis/ready-to-submit-completed.page.js'
+  import Navigation from '../../../pages/surveys/ukis/navigation.page.js'
+
+  const expect = chai.expect
+
+  describe('UKIS - Should the description be UKIS - Yes to 2.1, 3.1, 3.4, 3.6, 3.9, 3.11, 3.13, 3.15, 4.1, 4.3, 5.1 6.1 skips to interstitial section 6', function() {
+
+    it('Given I have selected yes to ANY of questions 2.1, 3.1, 3.4, 3.6, 3.9, 3.11, 3.13, 3.15, 4.1, 4.3, 5.1 6.1, When I do not answer 6.13, Then I will be routed to 6 interstitial', function() {
+      openQuestionnaire('1_0001.json')
+      Navigation.navigateToInnovationActivitiesBusinessStrategyandPractices()
+      BusinessChanges.clickBusinessChangesBusinessPracticesAnswerYes()
+        .clickBusinessChangesOrganisingAnswerYes()
+        .clickBusinessChangesExternalRelationshipsAnswerYes()
+        .clickBusinessChangesAnswerYes()
+        .submit()
+      Navigation.navigateToConstraintsonInnovation()
+      ConstraintsInnovationActivities.submit()
+      ConstrainingInnovationEconomic.submit()
+      ConstrainingInnovationCosts.submit()
+      ConstrainingInnovationFinance.submit()
+      ConstrainingInnovationAvailableFinance.submit()
+      ConstrainingInnovationLackQualified.submit()
+      ConstrainingInnovationLackTechnology.submit()
+      ConstrainingInnovationLackInformation.submit()
+      ConstrainingInnovationDominated.submit()
+      ConstrainingInnovationUncertain.submit()
+      ConstrainingInnovationGovernment.submit()
+      ConstrainingInnovationEu.submit()
+      ConstrainingInnovationReferendum.submit()
+      ConstraintsOnInnovationCompleted.submit()
```

Comment:
The then statement suggests we need to check that we appear on this page and not the next. This applies to all below